charts/ndm/templates/02-daemonset.yaml

```diff
@@ -0,0 +1,148 @@
+{{- if .Values.ndm.enabled }}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ template "ndm.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "ndm.name" . }}
+    chart: {{ template "ndm.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    component: ndm
+    openebs.io/component-name: ndm
+    openebs.io/version: {{ .Values.release.version }}
+spec:
+  updateStrategy:
+    type: "RollingUpdate"
+  selector:
+    matchLabels:
+      app: {{ template "ndm.name" . }}
+      release: {{ .Release.Name }}
+      component: ndm
+  template:
+    metadata:
+      labels:
+        app: {{ template "ndm.name" . }}
+        release: {{ .Release.Name }}
+        component: ndm
+        openebs.io/component-name: ndm
+        name: openebs-ndm
+        openebs.io/version: {{ .Values.release.version }}
+    spec:
+      serviceAccountName: {{ template "ndm.serviceAccountName" . }}
+      hostNetwork: true
```

Comment:
`hostPID` also needs to be added. Ref [here](https://github.com/openebs/openebs/blob/91b3968463dea4df861c7f2a15725a17624e4f35/k8s/openebs-operator.yaml#L480). It should be added based on the feature gate APIService.

Reply:
done